docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -1,15 +1,17 @@
 ---
 slug: /feature-flag-quickstart
-sidebar_position: 3
+sidebar_position: 2
 ---
 
-# Your first feature flag
+# Your first experiment
```

Comment:
I am not a big fan of this change in naming, I think it makes the nav items very confusing:

<img width="290" alt="Screenshot 2024-01-31 at 4 49 35 PM" src="https://github.com/Eppo-exp/eppo-docs/assets/3537793/91874e94-5d82-4bfc-9a92-0351c737fede">

Reply:
What do you think about this:

- Launching an experiment 
- Creating a metric
- Analyzing an experiment

I think the content in "Initial setup" could be rolled into the other sections

Reply:
Agree with Sven it's confusing as is. "Launching" or "Deploying" an experiment is better.